Applies fixes for Doc Parser. (#1098)
enso-org/ide: CHANGELOG.md

This is a minor update exposing more control over the anonymous data collection. You are now allowed 
to run the application with `--no-errors-data-collection`, and `--no-data-collection`. The options 
disable the collection of errors data (keeping the collection of other events), and disables the 
collection of all data, respectively.
  
To learn more what data is collected in Enso alpha releases, 

> read the notes of the `Enso 2.0.0-alpha.1` release.

Reviewer: This line is strangely wrapped. Please wrap it near the spot where other lines are wrapped.

> read the notes of the `Enso 2.0.0-alpha.1` release.

Reviewer 2: Put that pls in the `bug` fixes section (according to the CHANGELOG_TEMPLATE.md template - with link to your PR).

  now on all supported platforms.
- [You can now see data frames in the table visualisation][1181]. Big tables get truncated to 2000 
  entries.

> - [Fixing documentation layout in Searcher][1098]. Includes misinterpreted tags and unknown characters that are commonly used.

Reviewer 2: the changelog entries should be understandable to the end-users. Not all are perfect now, but this description "fixing documentation layout in Searcher" is not very understandable. Also, "fixing" means that it is being done, not done already, so its not a good form for a changelog. Can we describe here what was fixed and what kind of docs will be displayed better now ?

  now on all supported platforms.
- [You can now see data frames in the table visualisation][1181]. Big tables get truncated to 2000 
  entries.

> - [Fixing documentation layout in Searcher][1098]. Hotfixes for things such as misinterpreted tags 
>   and unknown characters that are commonly used.

Reviewer: ```suggestion
- [Fixing documentation layout in Searcher][1098]. Includes misinterpreted tags and unknown characters that are commonly used.
```

  now on all supported platforms.
- [You can now see data frames in the table visualisation][1181]. Big tables get truncated to 2000 
  entries.
- [Updated documentation parser][1098] with hotfixes for things such as misinterpreted tags or 

>   unknown characters that are commonly used.

Reviewer: I'm not sure if user will know what is "documentation parser". Let's name it "Fixing documentation layout in Searcher" as this is currently the visible change.
